example/example-Tzip16OnChainOne.ts

```diff
@@ -3,7 +3,7 @@ import { importKey } from '@taquito/signer';
 import { contractCode, metadataViewsExample1 } from '../integration-tests/data/metadataViews';
 import { char2Bytes } from '@taquito/utils';
 
-const provider = 'https://hangzhounet.api.tez.ie';
+const provider = 'http://ecad-ithacanet-full.i.tez.ie:8732'
```

Comment:
Should we use the public endpoint `https://ithacanet.ecadinfra.com/` instead of `http://ecad-ithacanet-full.i.tez.ie:8732` so the examples are available to the users?  
Same for the other examples.

Reply:
changed the address